SQL/0000-00-00-schema.sql

```diff
@@ -1643,6 +1671,391 @@ CREATE TABLE `parameter_type_override` (
   PRIMARY KEY (`Name`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+-- ********************************
+-- biobank tables
+-- ********************************
+                                                                                
+-- CREATES --
+
+/*Global*/
+
+CREATE TABLE `biobank_unit` (
+  `UnitID` integer unsigned NOT NULL AUTO_INCREMENT,
+  `Label` varchar(20) NOT NULL UNIQUE,
+  CONSTRAINT `PK_biobank_unit` PRIMARY KEY (`UnitID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+
+/*Container*/
```

Comment:
Consider, explaining that the a container is not just a container in the code but also real life.